analyzer.go

```diff
@@ -1,50 +1,54 @@
 package lifecycle
 
 import (
-	"fmt"
-	"path/filepath"
-
 	"github.com/buildpacks/imgutil"
 	"github.com/pkg/errors"
 
+	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/buildpack"
-	"github.com/buildpacks/lifecycle/launch"
+	"github.com/buildpacks/lifecycle/cmd"
 	"github.com/buildpacks/lifecycle/platform"
 )
 
 type Analyzer struct {
-	Buildpacks []buildpack.GroupBuildpack
-	LayersDir  string
-	Logger     Logger
-	SkipLayers bool
+	Image         imgutil.Image
```

Comment:
I love putting this (and `Cache`) on the struct!

Reply:
Could we add a comment to indicate which fields are required for which api?

Reply:
We may want to consider either alphabetizing these, or sequencing them in some sort of logical order.